internal/resources/dynamodb.go

```diff
@@ -82,12 +82,19 @@ func (ddb *DynamoDb) GetAll() ([]*ResourceSummary, error) {
 			ttlEnabled = true
 		}
 
+		avg_item_size := int64(0)
+		if *dRsp.Table.ItemCount != 0 {
+			avg_item_size = *dRsp.Table.TableSizeBytes / *dRsp.Table.ItemCount
```

Comment:
The struct doesn't have any explicit "avg item size". This is really what the DDB console also does underneath.